src/views/user/card.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Link} from 'react-router';
+const cardStyle= {
+  flex: 1,
+  minWidth: '300px',
+  maxWidth: '300px',
+  marginTop: '7px',
+  marginRight: '20px',
+  marginBottom: '20px'
+};
+
+const Card = ({title,description, route, destination }) => {
```

Comment:
why don't you use the Panel component ?

https://github.com/get-focus/focus-components/tree/develop/src/panel